components/layout/Layout.tsx

```diff
@@ -85,6 +93,32 @@ const SideNavWrapper = styled.div`
   flex-direction: column;
 `;
 
+const StyledCloseIcon = styled.div`
+  cursor: pointer;
+  svg {
+    * {
+      fill: ${ORANGE_PRIMARY};
+    }
+  }
+
+  :hover,
+  &.isFocused {
+    svg {
+      * {
+        fill: ${GREY_8};
+      }
+    }
+  }
```

Comment:
This works as a placeholder hover state fr Close icon, but I'm open to any suggestions on what styling to use for Hover states.